### PR TITLE
Fix/alfa 373 nodes saved language

### PR DESCRIPTION
**Pull request by MohamedTaha880**

### CODE REVIEW

File: src/modules/environment/hooks/environment.hook.inc [hunk 1]
```diff
@@ -33,7 +33,9 @@ function environment_doctor(Environment $env, $vars) {
 }
 
 function environment_boot(Environment $env, $vars){
-  $envFilePath= $env->dir['env'];
+  $full_host = strtolower($_SERVER['HTTP_HOST']);
+  $host = explode(':', $full_host)[0];
+  $envFilePath= $_SERVER['DOCUMENT_ROOT']. '/sites/'.$host.'/.env';
```
Comment (Aldus83) [inline src/modules/environment/hooks/environment.hook.inc]: You already should have this variable valorised by Quanta (I think it's called $env->dir['docroot']) no need to reimplement, better using standard variables

**Review by Aldus83 — CHANGES_REQUESTED** [review]
just an amendment to not reimplement existing dir variables

File: src/modules/environment/hooks/environment.hook.inc (hunk 1 above)
Comment (MohamedTaha880) [reply src/modules/environment/hooks/environment.hook.inc]: @Aldus83 
Done

**Comment by Aldus83** [comment]
@MohamedTaha880  please resolve conflict, we need to merge this

**Pull request merged**